draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -917,6 +917,13 @@ Example use cases include:
    all packets can be scheduled  over all paths leading to high capacity and
    and high energy efficiency.
 
+The default behavior is, that a path can always be used for packet scheduling 
+decisions (MP_PRIO=3), if the path has been established and added to an 
+existing MP-DCCP. Setting all paths to not use (MP_PRIO=0) MUST be avoided. 
+MP_PRIO is assumed to be exchanged reliably using the MP_CONFIRM mechanisms 
+(see {{ref-mp-option-confirm}}). 
+
+
 [TBD/TBV also consider rate limiting a given path]
```

Comment:
Thinking of rate limitation, for me the questions are whether in addition relative values (e.g., max. 40% of total traffic) or absolute (max. x kbps) via this path should be allowed, and how to reflect in MP_PRIO (should then maybe replace always for 3?) or whether an additional option has to be specified allowing for limiting several but one path?
If it is only limited to a fixed (e.g., less than 50%) then a new example use case should be added, as e.g.:
4) Setting Wi-Fi path to Always and Cellular path to Rate-Limited (since cellular
   here is the more constraint/valuable resource shared with more concurrent 
   users). In this case, maximum half of all packets can be scheduled over the 
   cellular paths avoiding draining/too heavy load in cellular network (e.g., in 
   case of hybrid 5G-RG as UE).

Reply:
I think having rate limiting in the protocol is perhaps overkill? A fraction of the traffic does not say anything about the amount of traffic that will be transferred unless you also know the sending rate of the application, which is complicated. Setting a max rate for how much is allowed over a path should perhaps better be set outside MP-DCCP if that is what is needed. The effect of this on the application is unknown in both cases unless you know the traffic profile. 

I would go for simplicity and leave it out for now. Otherwise the interaction with MP_PRIO would also need to be detailed as you mention @dirkhugo. Adding it externally or through the scheduling policy seems simpler to me.

Reply:
i think adding rate control requires more complex configuration and I think a new primitive, maybe we should not open up this now but ask on interim to provide comments/suggestions.

Reply:
@dirkhugo, I share @abrunstrom view. Leave it for now to reduce complexity. If really required it can be solved within a scheduler logic.

Reply:
I agree with you that we should go for basic operation in v1 ... just wanted to address the note and start discussion :-)